### ISSUE TITLE
ISSUE 1

### ISSUE THREAD

**Issue by KorryKatti** [KorryKatti/Python-Image-Board#2]
1. Make the global image board
2.  Make a header which displays the first three letters of each site as hyperlinks

these two are issues which should be solvable by beginners we will move to more advanced soon

You can use your own design for the global image board. No need to follow the dark theme on there but is recommended.

Comment below to be assigned to the issue.

**Comment by KorryKatti** [comment]
preview available at [korrykatti.pythonanywhere.com](https://korrykatti.pythonanywhere.com)

**Comment by Abhinavexists** [comment]
i want to try this issue

**Comment by lubdha-chaudhari** [comment]
I would like to contribute for this issue

**Comment by KorryKatti** [comment]
ok i will assign to you both , please make the pull requests only after checking your code

**Comment by NaMiKA0305** [comment]
can i contribute this type of webpages here

**Comment by KorryKatti** [comment]
> can i contribute this type of webpages here

please read the issue to know what is required right now , if your code aligns with it you are more than welcome

**Comment by Abhinavexists** [comment]
I have done the changes @KorryKatti , can you review please

**Comment by KorryKatti** [comment]
> I have done the changes @KorryKatti , can you review please

alright checking

**Comment by Shouryan-Bharote** [comment]
I would like to work on this issue
since i see someone has already solved it
please tell if there is another issue

**Comment by KorryKatti** [comment]
> I would like to work on this issue since i see someone has already solved it please tell if there is another issue

you will work on python backend or html fronted ?

**Comment by Shouryan-Bharote** [comment]
> > I would like to work on this issue since i see someone has already solved it please tell if there is another issue
> 
> you will work on python backend or html fronted ?
shall i optimize the code?

**Comment by KorryKatti** [comment]
> > > I would like to work on this issue since i see someone has already solved it please tell if there is another issue
> > 
> > 
> > you will work on python backend or html fronted ?
> > shall i optimize the code?

someone is already working on adding a new imageboard at #6 , so if you can complete making the backend for the global imageboard before them your PR will be accepted

**Comment by KorryKatti** [comment]
THIS IS NOW COMPLETE
